test/uk/gov/hmrc/residencenilratebandcalculator/json/JsonBuilderSpec.scala

```diff
@@ -359,6 +359,25 @@ class JsonBuilderSpec extends UnitSpec with MockitoSugar with Matchers with With
         }
       }
 
+      "the CacheMap contains a date of disposal prior to the eligibility date for DateOfDisposal" in {
+        val cacheMap = CacheMap(id = cacheMapId, data =
+          Map(
+            Constants.chargeableTransferAmountId -> JsNumber(100),
+            Constants.dateOfDeathId -> JsString("2018-01-01"),
+            Constants.dateOfDisposalId -> JsString("2015-01-01"),
+            Constants.grossEstateValueId -> JsNumber(200),
+            Constants.propertyValueId -> JsNumber(200),
+            Constants.percentageCloselyInheritedId -> JsNumber(50)))
+        val buildResult = await(jsonBuilder.buildFromCacheMap(cacheMap))
+        buildResult match {
+          case Failure(exception) => {
+            exception shouldBe a [JsonInvalidException]
+            exception.getMessage shouldBe "JSON error: Date of disposal is before eligibility date\n"
+          }
+          case Success(json) => fail
+        }
+      }
+
```

Comment:
Because we don't need to add the date of disposal to the JSON (yet!), we can remove these changes to the tests